frodo.js

```diff
@@ -8,33 +8,31 @@
  *
```

Comment:
SCRIPT-This block of code could benefit from some comments explaining the logic behind it.